### PR TITLE
Magento 2.3.4 Release Notes

### PR DESCRIPTION
**Pull request by jfrontain**
This pull request (PR) adds the GA version of the Magento 2.3.4 release notes to the doc set. It contains all final review comments.

### CODE REVIEW

File: src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md [hunk 1]
```diff
@@ -9,11 +9,13 @@ This release includes over 220 functional fixes to the core product and  over 30
...
+Merchants can now install time-sensitive security fixes without applying the hundreds of functional fixes and enhancements that a full quarterly release (for example, Magento 2.3.4) provides. Patch 2.3.3.1 (Composer package 2.3.3-p1) is a security-only patch that provides fixes for vulnerabilities that have been identified in our previous quarterly release, Magento 2.3.3. All hot fixes that were  applied to the 2.3.3 release are included in this security-only patch. (A *hot fix* provides a fix to a released version of Magento that addresses a specific problem or bug.) For general information about security-only patches, see the Magento DevBlog post [Introducing the New Security-only Patch Release](https://community.magento.com/t5/Magento-DevBlog/Introducing-the-New-Security-only-Patch-Release/ba-p/141287). For instructions on downloading and applying security-only patches (including patch 2.3.3-p1), see [Install Magento using Composer](https://devdocs-beta.magento.com/guides/v2.3/install-gde/composer.html#get-the-metapackage). Security-only patches include only security bug fixes, not the additional security enhancements that are included in the full patch.
```
Comment (dobooth) [inline src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md]: [Install Magento using Composer](https://devdocs-beta.magento.com/guides/v2.3/install-gde/composer.html#get-the-metapackage)

File: src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md [hunk 2]
```diff
@@ -116,7 +120,9 @@ For information on these enhancements plus other improvements, see [PWA Studio r
 
 ### dotdigital
 
-Live Chat powered by dotdigital enables merchants to increase conversion rates, and keep customers coming back with real-time engagement. All Magento 2.3.x merchants (both Magento Open Source and Magento Commerce) can receive a free live chat agent without the need for a full dotdigital Engagement Cloud license.
+*  Live Chat powered by dotdigital enables merchants to increase conversion rates, and keep customers coming back with real-time engagement. All Magento 2.3.x merchants (both Magento Open Source and Magento Commerce) can receive a free live chat agent without the need for a full dotdigital Engagement Cloud license.
+
+*  Engagement Cloud includes a new Chat widget that makes it easy for shoppers to communicate in real time with customers as they shop in your store. Chat can be accessed from the Engagement Cloud section of the Magento configuration, or directly from your Engagement Cloud account. See [Engagement Cloud Chat](https://docs.magento.com/m2/ee/user_guide/marketing/engagement-cloud-chat.html).
```
Comment (dobooth) [inline src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md]: Is it '...shoppers to communicate in real time with customers as...'  or '...merchants to communicate....'?

File: src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md [hunk 3]
```diff
@@ -284,7 +294,7 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 <!--- MC-19836-->
 
-*  The **Select All** checkbox on the Requisition List page now works as expected. Previously, when you deselected all products in a requisition list, the **Select All** checkbox remained enabled.
+*  The **Select All** checkbox on the Requisition List page now works as expected. Previously, when you deselected all products in a Requisition Lis, the **Select All** checkbox remained enabled.
```
Comment (dobooth) [inline src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md]: List

File: src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md [hunk 4]
```diff
@@ -1750,7 +1766,7 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 <!--- MC-19414-->
 
-*  The Low Stock report now accurately displays low stock levels. Previously,  this report showed a product level of 0 even when product stock exceeded 0.
+*  The Low Stock report now accurately displays low stock levels. Previously,  this report showed a product level of zero (0) even when product stock exceeded zero.
```
Comment (dobooth) [inline src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md]: Double space after Previously, ?

File: src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md [hunk 5]
```diff
@@ -1816,11 +1832,11 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 <!--- MC-17014-->
 
-*  User can now successfully save a fixed product tax  (FPT) to a product that is assigned to a specific website.
+*  You can now successfully save a fixed product tax  (FPT) to a product that is assigned to a specific website.
 
 <!--- ENGCOM-5919-->
 
-*  Inconsistent sorting of FPT and tax totals has been resolved on the Admin order, create invoice, invoice, create credit memo,  and credit memo pages. *Fix submitted by Mahesh Singh in pull request [24663](https://github.com/magento/magento2/pull/24663)*. [GitHub-24225](https://github.com/magento/magento2/issues/24225)
+*  Inconsistent sorting of fixed product tax  (FPT and tax totals has been resolved on the Admin order, create invoice, invoice, create credit memo,  and credit memo pages. *Fix submitted by Mahesh Singh in pull request [24663](https://github.com/magento/magento2/pull/24663)*. [GitHub-24225](https://github.com/magento/magento2/issues/24225)
```
Comment (dobooth) [inline src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md]: A couple double spaces: `tax  (` and `memo,  and`.  One more above on line 1825: tax . (

File: src/guides/v2.3/release-notes/release-notes-2-3-4-open-source.md [hunk 6]
```diff
@@ -1508,11 +1510,11 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 <!--- MC-17014-->
 
-*  User can now successfully save a fixed product tax  (FPT) to a product that is assigned to a specific website.
+*  You can now successfully save a fixed product tax  (FPT) to a product that is assigned to a specific website.
```
Comment (dobooth) [inline src/guides/v2.3/release-notes/release-notes-2-3-4-open-source.md]: Same dupe spaces here and next line.

File: src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md (hunk 1 above)
Comment (jfrontain) [reply src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md]: changed link in both source files

File: src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md (hunk 3 above)
Comment (jfrontain) [reply src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md]: changed in source file

File: src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md (hunk 5 above)
Comment (jfrontain) [reply src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md]: Thank you, but no time to groom for double spaces that do not affect meaning or display.

File: src/guides/v2.3/release-notes/release-notes-2-3-4-open-source.md (hunk 6 above)
Comment (jfrontain) [reply src/guides/v2.3/release-notes/release-notes-2-3-4-open-source.md]: Thank you, but no time to groom for double spaces that do not affect meaning or display.

File: src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md (hunk 4 above)
Comment (jfrontain) [reply src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md]: Thanks, but not necessary

File: src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md (hunk 2 above)
Comment (jfrontain) [reply src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md]: Not changing this text as the writer and their developer suggested this wording,

**Comment by jfrontain** [comment]
running tests

**Comment by jfrontain** [comment]
running tests

**Pull request merged**